### PR TITLE
fix Mailru mapUserToObject id attribute

### PR DESCRIPTION
**Pull request by Cellard**

### CODE REVIEW

File: src/Mailru/Provider.php [hunk 1]
```diff
@@ -53,7 +53,7 @@ protected function getUserByToken($token)
     protected function mapUserToObject(array $user)
     {
         return (new User())->setRaw($user)->map([
-            'id'       => $user['email'],
+            'id'       => $user['id'],
```
Comment (atymic) [inline src/Mailru/Provider.php:56]: this would be a breaking change i guess?

Comment (Cellard) [reply src/Mailru/Provider.php:56]: Likely. But it is anyway broken for now (

Email is not an identifier. And MailRu provides an identifier.

**Review by atymic — APPROVED** [review]

**Comment by Cellard** [comment]
@atymic I forget to fix README for the provider. Should I make new PR or maybe you fix docs yourself?

### Returned User fields

- ``id`` ~~(same as ``email``)~~
- ``nickname``
- ``name``
- ``email``
- ``avatar``

**Comment by atymic** [comment]
Fixed :)

**Pull request merged**